Class-Examples/XML/emilyAssignment3.xml

```diff
@@ -0,0 +1,202 @@
+                    wrench, turning clockwise, as much as you can. Push down on the lug wrench with
+                    the full weight of your body.</body>
+            </step>
+            <step num="14"><heading xml:id="lowerCompletely">LOWER THE VEHICLE COMPLETELY</heading>
+                <body><action>Bring</action> the vehicle all the way to the ground and
+                        <action>remove</action> the <item xml:id="jack">jack</item>. Give the <item
+                        xml:id="lugNut">lug nuts</item> another pull with the wrench to
+                        <action>ensure</action> they’re as tight as possible</body>.</step>
+            <step num="15">
+                <heading xml:id="replaceHubcap">REPLACE THE HUBCAP</heading>
+                <body>If the <item xml:id="hubCap">hubcap</item> you took from the <item
+                        xml:id="flatTire">flat tire</item> will fit your spare, <action>put it in
+                        place</action> the same way you removed it initially. If it doesn’t fit,
+                    stow it away with the tire when you stow your equipment.</body>
+            </step>
+            <step num="16">
+                <heading xml:id="stowEqipment">STOW ALL EQUIPMENT</heading>
+                <body>You have before you a <item xml:id="jack">jack</item>, a <item
+                        xml:id="lWrench">lug wrench</item>, <item xml:id="whWedges">wheel
+                        wedges</item>, your <item xml:id="flatTire">flat tire</item>, and possibly a
+                        <item xml:id="hubCap">hubcap</item>. Don’t forget to put all of them in your
+                    vehicle before driving away.</body>
+            </step>
+            <step num="17">
+                <heading xml:id="checkSparePressure">CHECK THE PRESSURE IN THE SPARE TIRE</heading>
+                <body>You should <action>check</action>the tire pressure of the spare tire to make
+                    sure that it is safe to drive on. “T-Type” temporary spares, also called
+                    “mini-spares,” require 60 psi (420 kPa). If the tire needs pressure, drive
+                    (slowly) to a service station immediately.</body>
+            </step>
+            <step num="18">
+                <heading xml:id="flatToTechnician">TAKE YOUR FLAT TIRE TO A TECHNICIAN</heading>
+                <body>Temporary spare tires aren’t made to drive long distances or at high speeds,
+                    so drive cautiously until you’re able to visit a tire technician. A professional
+                    should be able to determine whether your tire needs a repair or if it’s time to
+                    replace it.</body>
+            </step>
+        </body>
+    </para>
+    <para num="7"><heading>HOW LONG DOES IT TAKE TO FIX A FLAT TIRE? </heading>Aside from taking
+        your tire to a professional, the above procedure shouldn’t take more than
+                <time><quant>15</quant> to <quant>30</quant>
```

Comment:
@epw5194 This reminds me of the recipe coding we worked on, capturing quantities and units in code. Nicely done.